templates/macros.html.tera

```diff
@@ -18,6 +18,7 @@
                 link_4="</a>")
             }}
             ⋅ {{ article.date | date(format="%B %e") }}
+            ⋅ <a href="{{article.blog_url}}">{{article.blog.title}}</a>
```

Comment:
About the code style: if you could add spaces between the braces and the expression they contain, it would be more readable (and consistent with what was done before).